VRDR/DeathRecord.cs

```diff
@@ -194,7 +194,7 @@ public DeathRecord()
             DeathCertification = new Procedure();
             DeathCertification.Id = Guid.NewGuid().ToString();
             Identifier certificationIdentifier = new Identifier();
-            certificationIdentifier.Value = "";
+            certificationIdentifier.Value = "000000";
```

Comment:
In Canary, in the Connectathon record tests, if a user provides a record that has no certificate number Canary will report

```
Expected Certificate number: 00051
Found value: 000000
```
and

```
JSON

Expected:

{"identifier": "[
  {
    "value": "000051"
  }
]"}

How Canary interpreted your input:

{"identifier": "[
  {
    "value": "000000"
  }
]"}
```

Seems like this would be confusing...

Reply:
Why does Canary not use exactly what the tester submitted rather than ingesting then recreating it (with possible default values like this)?

Reply:
On further consideration, I think the entire Identifier should be left out. The things in the constructor are bare minimums for referencing, and this Identifier isn't necessary for that.

Reply:
The profile requires 1..* identifier on that element, if we leave it out then our default would be to generate an invalid entry.

Reply:
Which brings us back to the approach I suggested of adding the identifier but not including the value at all, which is valid. Any reasons not to do that?

Reply:
An empty identifier value is invalid.

Reply:
Are we promising that just doing a `new DeathRecord()`, without including anything else, will return something that is "validator valid"? Is that the reason for including this in the first place?

Reply:
Good question. At a minimum I think we'd want it to be valid FHIR but we probably don't want to promise compatibility with the VRDR or US Core profiles since that would require us to include a lot of default data. I'm leaning toward not including the identifier at all (valid FHIR, not valid against against VRDR) vs including an empty identifier (causes parser errors).

Reply:
By "not including the value at all", I don't mean including a blank string, I mean not having a value field present in the identifier. AFAICT that's valid.

Reply:
`identifier: [ {} ]` causes a parser error, identifier: [] is ok.

Reply:
Is that what we get if we simply remove the line of code

```
certificationIdentifier.Value = <something>;
```

?

Reply:
We'd also need to see how Canary handles that...